init-app.sh

```diff
@@ -104,20 +104,12 @@ eval set -- "$PARAMS"
 oc login -u ${OPENSHIFT_USER} https://${OPENSHIFT_HOST}:${OPENSHIFT_PORT}
 oc project ${OPENSHIFT_PROJECT}
 
-# ensure we have imagestreams for our build dependencies.
-OUT=$(oc get -n ${OPENSHIFT_PROJECT} -o yaml is postgresql 2>/dev/null | grep -c 'tag: "9\.6"')
-if [ $OUT != 1 ]; then
-    oc create -n ${OPENSHIFT_PROJECT} istag postgresql:9.6 --from-image=centos/postgresql-96-centos7
-fi
-
-OUT=$(oc get -n ${OPENSHIFT_PROJECT} -o yaml is python 2>/dev/null | grep -c 'tag: "3\.6"')
-if [ $OUT != 1 ]; then
-    oc create -n ${OPENSHIFT_PROJECT} istag python:3.6 --from-image=centos/python-36-centos7
-fi
+oc apply -f openshift/koku-template.yaml
```

Comment:
Should we use the variable here instead of hard coding?

Reply:
good catch. yes, here.